hal/src/common/timer_traits.rs

```diff
@@ -0,0 +1,12 @@
+use crate::hal::timer::{CountDown, Periodic};
+use crate::time;
+
+/// Trait for timers that can enable & disable an interrupt the fires
```

Comment:
nit: `enable & disable an interrupt that fires`

Reply:
fixed